pkg/controller/cincinnati/cincinnati_controller.go

```diff
@@ -150,6 +150,21 @@ func (r *ReconcileCincinnati) Reconcile(request reconcile.Request) (reconcile.Re
 			Message: "",
 		})
 	}
+	if instance.Spec.GraphDataImage != "" && (instance.Spec.GitHubOrg != "" || instance.Spec.GitHubRepo != "" || instance.Spec.Branch != "") {
+		conditionsv1.SetStatusCondition(&instanceCopy.Status.Conditions, conditionsv1.Condition{
+			Type:    cv1alpha1.ConditionConfigurationConflict,
+			Status:  corev1.ConditionTrue,
+			Reason:  "ConflictFound",
+			Message: "graphDataImage and gitHubOrg/gitHubRepo/branch should not be configured at the same time.",
```

Comment:
Should also mention that the `graphDataImage` is winning.

Reply:
Done.